AmazonChimeSDK/AmazonChimeSDK/internal/video/DefaultVideoClientController.swift

```diff
@@ -521,13 +519,13 @@ extension DefaultVideoClientController: VideoClientController {
 
     public func subscribeToReceiveDataMessage(topic: String, observer: DataMessageObserver) {
         if dataMessageObservers[topic] == nil {
-            dataMessageObservers[topic] = ConcurrentMutableSet()
+            dataMessageObservers[topic] = NSMutableSet()
         }
         dataMessageObservers[topic]?.add(observer)
     }
 
     public func unsubscribeFromReceiveDataMessageFromTopic(topic: String) {
-        dataMessageObservers.removeValue(forKey: topic)
+        dataMessageObservers[topic] = nil
```

Comment:
I think the entry will be kept in the `dataMessageObservers` by setting it to nil, it will be more appropriate to implement `remove()` in `ConcurrentDictionary`.

Reply:
This is fine. We are overriding subscript setting and using a lock: https://github.com/aws/amazon-chime-sdk-ios/blob/master/AmazonChimeSDK/AmazonChimeSDK/internal/utils/ConcurrentDictionary.swift#L24. So the internal dictionary gets updated. Apple: "If you assign nil to an existing key, the key and its associated value are removed".